outputs/email.go

```diff
@@ -13,10 +13,40 @@ import (
 	"github.com/aquasecurity/postee/v2/layout"
 )
 
+type EmailSender interface {
+	Send(address string, to []string, body []byte) error
+}
+
+type EmailConfig struct {
```

Comment:
I don't see why properties of Email Output can't be used. If it required for testing then
Just defining `send`function  and then override it (and keep original code) would be enough. Duplicated properties require extra efforts on modification.

Reply:
You're right, I've simplified it. thanks!